File: app/src/main/java/com/thesohelshaikh/ytanalyser/VideoModel.java
package com.thesohelshaikh.ytanalyser;

public class VideoModel {
    String id;
    String duration;

    public VideoModel(String id, String duration) {
        this.id = id;
        this.duration = duration;
    }

    public String getId() {
        return id;
    }

    public void setId(String id) {
        this.id = id;
    }

    public String getDuration() {
        return duration;
    }

    public void setDuration(String duration) {
        this.duration = duration;
    }

    @Override
    public String toString() {
        return "VideoModel{" +
                "id='" + id + '\'' +
                ", duration='" + duration + '\'' +
                '}';
    }
}

File: app/src/main/java/com/thesohelshaikh/ytanalyser/model/VideoModel.kt
package com.thesohelshaikh.ytanalyser.model

class VideoModel {
    var id: String? = null
    var duration: String? = null
    var title: String? = null
    var thumbnailURL: String? = null
    var channelTitle: String? = null

    constructor() {}
    constructor(
        id: String?,
        duration: String?,
        title: String?,
        thumbnailURL: String?,
        channelTitle: String?
    ) {
        this.id = id
        this.duration = duration
        this.title = title
        this.thumbnailURL = thumbnailURL
        this.channelTitle = channelTitle
    }

    override fun toString(): String {
        return "VideoModel{" +
                "id='" + id + '\'' +
                ", duration='" + duration + '\'' +
                '}'
    }
}